docs/docs-content/vm-management/create-manage-vm/advanced-topics/migrate-vm-kubevirt.md

```diff
@@ -0,0 +1,118 @@
+---
+sidebar_label: "Migrate a VM to a VMO cluster"
+title: "Migrate a VM to a VMO cluster"
+description: "Learn how to migrate VMs to Palette VMO using the Palette CLI."
+icon: " "
+hide_table_of_contents: false
+sidebar_position: 40
+tags: ["vmo", "palette-cli"]
+---
+
+During large scale Kubernetes adoptions, workloads are often rehosted or migrated instead of being redeployed from
+scratch. This process allows system administrators to copy the application, together with its data, to a Kubernetes
+cluster. This process can be time consuming if done manually, so it is often automated with open-source tools such as
+[Forklift](https://github.com/kubev2v/forklift).
+
+The [Palette CLI](../../../automation/palette-cli/palette-cli.md) provides the ability to migrate Virtual Machines (VMs)
+from VMware vSphere to Palette VMO.
+
+## Limitations
+
+- You can only migrate VMs hosted in VMware vSphere.
+
+## Prerequisites
+
+- A Healthy VMO cluster. Refer to the [Create a VMO Profile](../../create-vmo-profile.md) for further guidance.
+- One or more VMs hosted in VMware vSphere. The VMs should also operate one the
+  [`virt-v2v` supported guest systems](https://libguestfs.org/virt-v2v-support.1.html).
+  - The VMs must be powered off before migration.
+- The Palette CLI installed and setup. Refer to the
+  [Installation](../../../automation/palette-cli/install-palette-cli.md) guide for further details.
+  - The kubectl command-line tool should also be installed. Refer to the
+    [kubectl installation](https://kubernetes.io/docs/tasks/tools/install-kubectl/) guide to learn more.
+
```

Comment:
We should add vSphere credentials here :)

Reply:
Added.